src/Microsoft.Azure.WebJobs.Extensions.Kafka/Listeners/KafkaListener.cs

```diff
@@ -214,36 +214,14 @@ private ConsumerConfig GetConsumerConfiguration()
                 conf.SaslMechanism = SaslMechanism.Plain;
                 conf.SaslUsername = EventHubsSaslUsername;
                 conf.SaslPassword = this.listenerConfiguration.EventHubConnectionString;
-                conf.SslCaLocation= this.EnsureValidEventHubsCertificateLocation(this.listenerConfiguration.SslCaLocation);
+                conf.SslCaLocation= this.listenerConfiguration.SslCaLocation;
```

Comment:
Do we need to provide a CA for event hubs?

Reply:
Hi @fbeltrao 
Thank you for the review. I think we don't need CA anymore. not only EventHubs but others. 
We might be able to remove the SslCaLocation, however, there is leagacy code that cause breaking change by it and in some usecase still might use it. That is why I keep the `SslCaLocation` It's self.  Do you have any suggestion?

Reply:
Ok. Since [this](https://github.com/Azure/azure-functions-kafka-extension/blob/ef8b81f91da79c19b69967cdf0811fef10c1b29e/src/Microsoft.Azure.WebJobs.Extensions.Kafka/Listeners/KafkaListener.cs#L217) is inside the block for event hubs only and we don't need a CA we could remove the line, don't we?